CMakeLists.txt

```diff
@@ -206,6 +206,9 @@ set(PCL_OUTPUT_BIN_DIR "${PCL_BINARY_DIR}/${BIN_INSTALL_DIR}")
 make_directory("${PCL_OUTPUT_LIB_DIR}")
 make_directory("${PCL_OUTPUT_BIN_DIR}")
 if(WIN32)
+  set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${PCL_OUTPUT_LIB_DIR}")
+  set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${PCL_OUTPUT_BIN_DIR}")
+  set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${PCL_OUTPUT_BIN_DIR}")
```

Comment:
Notice that now we can move 
```cmake
  set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${PCL_OUTPUT_LIB_DIR}")
  set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${PCL_OUTPUT_BIN_DIR}")
```
out of the `if/else(WIN32)` conditionals, since if became common to all platforms.